package/python-kiwi-spec-template

```diff
@@ -123,16 +117,9 @@ Requires:       qemu-img
 Requires:       squashfs-tools
 Requires:       gdisk
 Requires:       dnf
+Provides:       kiwi-image:tbz
```

Comment:
I have doubts about this provides list, probably I'd put clearly together the requirement and the provides, so it is easy to spot which distro package build provides, requires and recommends each package manager.

Reply:
I put in this provides to the main package because it is actually true. The ability to create simple tarballs of the root tree is provided by the main package. The main package also has a requirement to tar set.

Currently this provides is used in "kiwi-image-tbz-requires" as it comes from kiwi-boot-descriptions. From a user perspective it does not make sense because the support for the "tbz" image type is provided by just installing python3-kiwi as this package already requires tar and nothing more is needed on the host to create tarballs from a root tree. This is the reason why I put this provides flag to the main package.

Not good ?

Reply:
The provides for `kiwi-image:tbz` is good. Sorry my comment was not clear enough and now I realized that it was under the wrong line. I wanted to comment to the  `kiwi-packagemanger:*` provides. Which have not really been modified by your PR. Due to the lack of context in the diff yesterday I got confused by those provides as I did not spot that they were already defined only defined for Fedora and RHEL.

I see the `kiwi-packagemanger:apt` is missing but IIRC there was an issue with the spec trying to add it next to the requirement of the debootstrap. Either dpkg or the spec to deb converter did not like having colons in the provides.